### PR TITLE
Stats: Reduxify fetching post likes

### LINKED ISSUE #10348: Stats: Add Likes Details to Summary Page
From pKdGS-1mh-p2

A possible enhancement for the Post Stats Summary page is inclusion of likes data for the given post.  We currently show like totals in the post list card:

blog_posts_ _the_wordpress_com_blog_ _wordpress_com

The user suggestion was to display gravatars of the post likers ( likees ? ).  Perhaps we could also show the number of comments on the post too.

### PR DESCRIPTION
**Pull request by youknowriad**
As a first step to resolve #10348 I'm creating a `likes` state subtree under `posts`.
This is also part of #5046 

Nothing to test for now.

### CODE REVIEW

File: client/state/posts/likes/test/reducer.js [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+			} );
+
+			expect( state ).to.eql( {
+				12345678: {
+					50: false
+				}
+			} );
+		} );
+
+		it( 'should set site ID to false if request finishes unsuccessfully', () => {
```
Comment (timmyc) [inline client/state/posts/likes/test/reducer.js]: Should this test description read `should set post ID to false if request finishes unsuccessfully`?

**Comment by youknowriad** [comment]
Good catch @timmyc I naively supposed all the likes will be returned. 
I guess this also answers the question of whether we should limit the avatars displayed since we would get a maximum of 90 in any circumstance.

**Comment by youknowriad** [comment]
updated in 4df3ac2 :)

**Comment by youknowriad** [comment]
I rebased the branch, do you think you'll have time to take a look again @timmyc ?

**Comment by timmyc** [comment]
Looking good now!  Thanks for updating that to accommodate all the data.

**Pull request merged**